tests/functional/monitoring/conftest.py

```
@@ -1213,3 +1213,102 @@ def teardown():
+                helpers.wait_for_resource_state(
+                    clone_pvc, constants.STATUS_BOUND, timeout=600
+                )
+            logger.info("All CSI clones are Bound")
+
+        run_time = 60 * 15
+        logger.info(
+            f"Waiting {run_time}s for HighRBDCloneSnapshotCount " "alert evaluation"
+        )
+        time.sleep(run_time)
+
+    def teardown():
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `teardown`

Add return type annotation: `None`

(ANN202)